style.css

```diff
@@ -71,5 +76,239 @@ h1 {
   background-image: url(image/image_geometry_2.jpg);
   background-repeat: no-repeat;
   background-size: 100%;
-  height: 610px;
+  height: 260px;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
+  margin-left: 5%;
+  font-size: 50px;
+}
+
+.Works-section {
+  display: grid;
+}
+
+.worksection1 {
+  background-image: url(image/Rectangle21.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection2 {
+  background-image: url(image/Rectangle22.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection3 {
+  background-image: url(image/Rectangle23.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection4 {
+  background-image: url(image/Rectangle24.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection5 {
+  background-image: url(image/Rectangle25.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.rectangle {
+  background-color: #1a2236;
+  height: 156px;
+}
+
+h3 {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  text-align: left;
+  font-size: 20px;
+  margin-left: 20px;
+}
+
+.buttons {
+  display: flex;
+  list-style: inside;
+  box-sizing: border-box;
+  margin-left: 1px;
+}
+
+.btn {
+  background-color: #98dede;
+  border-color: transparent;
+  width: auto;
+}
+
+.Ruby {
+  flex: 50%;
+}
+
+.HTMLCSS {
+  flex: 50%;
+}
+
+.Js {
+  flex: 50%;
+}
```

Comment:
- [ ] As the previous reviewer mentioned please make sure you sticked to the Figma design. you add padding for these elements.
# your design👇
![finishHtml](https://user-images.githubusercontent.com/66992901/191820262-ca538d4d-19b2-48b0-a1fa-4f7b85c65097.PNG)
#Figma design👇
![finishhtfigma](https://user-images.githubusercontent.com/66992901/191820341-a36755bc-1cc2-4e01-8f62-eb30442f2094.PNG)